.github/workflows/r-python-interop-testing.yml

```diff
@@ -66,15 +63,8 @@ jobs:
       #  if: ${{ matrix.os != 'macOS-latest' }}
       #  run: cd apis/r && Rscript -e "options(bspm.version.check=TRUE); install.packages('tiledb', repos = c('https://eddelbuettel.r-universe.dev/bin/linux/jammy/4.3/', 'https://cloud.r-project.org'))"
 
-      - name: Build and install libtiledbsoma
-        run: sudo scripts/bld --prefix=/usr/local --no-tiledb-deprecated=true --werror=true && sudo ldconfig
-
-      - name: Install R-tiledbsoma
```

Comment:
It's unclear to me why changing the R build is necessary for this PR.

I was suspecting you might have been pulling in something from another branch, I don't know what.

If you assert that these mods (lines 73-77 as of this writing) on R-related YAML are necessary for this Python/C++ PR to build, I will take that on your word. But I'd still be grateful for some information from you on what the build failure is: if we do need to modify R YAML for a Python/C++ PR, then I have something to learn, and I'd like to learn it from you.